From 804579b9f02916cd38d1e61952fbe852155caa5c Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Mon, 4 May 2020 15:09:20 +0000
Subject: [PATCH] Update newtonsoft json version for arcade
Arcade uses Newtonsoft.Json.  Source-build builds 2 versions, 9.0.1 and
and 12.0.2.  Arcade uses the older version, so the Version props
Version needs to be updated for Arcade.  This is planned to be
removed in Arcade in this issue: https://github.com/dotnet/arcade/issues/2763

---
 Directory.Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index eba8e1c..1cdf762 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -14,5 +14,6 @@
       Tools and packages produced by this repository support infrastructure and are not shipping on NuGet or via any other official channel.
     -->
     <IsShipping>false</IsShipping>
+    <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
   </PropertyGroup>
 </Project>
-- 
1.8.3.1

